left_hip_yaw:
  id: 4
  type: "xm430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

left_hip_roll:
  id: 5
  type: "xm430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

left_hip_pitch:
  id: 6
  type: "xm430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

left_knee:
  id: 7
  type: "mx64"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

left_ankle_pitch:
  id: 8
  type: "mx64"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

left_ankle_roll:
  id: 9
  type: "mx64"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

right_hip_yaw:
  id: 10
  type: "xm430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

right_hip_roll:
  id: 11
  type: "xm430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

right_hip_pitch:
  id: 12
  type: "xm430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

right_knee:
  id: 13
  type: "mx64"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

right_ankle_pitch:
  id: 14
  type: "mx64"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

right_ankle_roll:
  id: 15
  type: "mx64"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

left_shoulder_pitch:
  id: 18
  type: "mx28"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

left_shoulder_roll:
  id: 0
  type: "mx28"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

left_elbow:
  id: 1
  type: "mx28"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

right_shoulder_pitch: # Checked
  id: 19
  type: "mx28"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

right_shoulder_roll:
  id: 2
  type: "mx28"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

right_elbow:
  id: 3
  type: "mx28"
  angle_zero: 180
  angle_max: 360
  angle_min: 0

head_yaw:
  id: 16
  type: "xl430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
  flipped: "true"

head_pitch:
  id: 17
  type: "xl430"
  angle_zero: 180
  angle_max: 360
  angle_min: 0
